conf/config.properties.example

```diff
@@ -637,6 +637,9 @@ server.systemTest.runDestructive=false
 # Behavior without this config set to true is to give all users access to all projects
 server.project.roles.enable=true
 
+# Require project type to be returned by project customer specific CustomerSpecificFunctions
+server.project.type.required=false
+
```

Comment:
I wanted a way to know that we are in this mode.  This config is checked and an error is thrown if a project does not have a type returned by the service.